### DISCUSSION TITLE
Setup/

### DISCUSSION THREAD

**Discussion by giscus** [wmarti/MetalTutorial#1] [Announcements]
# Setup/

https://metaltutorial.com/Setup/

**Comment by davupls** [comment]
Not only am I happy to find a good tutorial on setting up Metal. But you also show me how to use GLFW using Xcode so I could use it for OpenGL later. Thanks a bunch.

**Comment by AlexanderDevaikinEnscape** [comment]
One issue happened on my machine: glfw could not be loaded due to signature issues. Installed via homebrew on Sequoia m1. The solution was to set it to be embedded and signed on the "General -> Frameworks and Libraries" project setup tab.

**Reply by sromic1990** [reply]
this was the exact same problem for me. Fixed it by your way. Thanks a lot!

**Comment by MilkGolium** [comment]
thanks very much 🙏

**Comment by Ajaya1000** [comment]
libglfw.dylib needs to be updated to "embed & sign"
